### PR TITLE
Fixed crash when moving type from System namespace

### PR DESCRIPTION
**Pull request by DoctorKrolic**
Fixes: https://github.com/dotnet/roslyn/issues/35507

### CODE REVIEW

File: src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs [hunk 1]
```diff
@@ -190,7 +190,7 @@ private static async Task<ImmutableArray<ISymbol>> GetMemberSymbolsAsync(Documen
             {
                 case TNamespaceDeclarationSyntax namespaceNode:
                     var containerSymbol = (INamespaceSymbol)semanticModel.GetDeclaredSymbol(container, cancellationToken);
-                    return containerSymbol.GetMembers().SelectAsArray(m => (ISymbol)m);
+                    return containerSymbol.GetMembers().Where(el => el.IsFromSource()).SelectAsArray(m => (ISymbol)m);
```
Comment (Youssef1313) [inline src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs:193]: I think this extension should be available to use:

https://github.com/dotnet/roslyn/blob/09408ab8a29e03caddfb11f29328c05169ac7cde/src/Compilers/Core/Portable/Collections/ImmutableArrayExtensions.cs#L199-L225

```suggestion
                    return containerSymbol.GetMembers().SelectAsArray(m => m.IsFromSource(), m => (ISymbol)m);
```

Comment (DoctorKrolic) [reply src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs:193]: This extension is for `ImmutableArray`, while `GetMembers()` returns `IEnumerable`. I should have checked it before accepting

Comment (Youssef1313) [reply src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs:193]: Interesting. It seems that `INamespaceSymbol` shadows `INamespaceOrTypeSymbol.GetMembers` (which returns ImmutableArray).

Comment (CyrusNajmabadi) [inline src/Features/Core/Portable/MoveToNamespace/AbstractMoveToNamespaceService.cs:193]: based on the code below, it seems like we want the syntactic members that are inside 'container'.  CAn you use syntaxFacts.GetMembersOfBaseNamespaceDeclaration like you do GetMembersOfCompilationUnit below?  Thanks!

**Comment by CyrusNajmabadi** [comment]
Can you add a test?  Thanks!

**Comment by DoctorKrolic** [comment]
Test for what? I've already added one

**Comment by CyrusNajmabadi** [comment]
Oh strange. GitHub had filtered them out. Thanks!

**Review by CyrusNajmabadi — APPROVED** [review]

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Pull request merged**